dspace-spring-rest/src/test/java/org/dspace/app/rest/EPersonRestRepositoryIT.java

```diff
@@ -54,32 +66,40 @@ public void findAllPaginationTest() throws Exception {
                                         .withEmail("Johndoe@gmail.com")
                                         .build();
 
-        getClient().perform(get("/api/eperson/eperson")
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(get("/api/eperson/epersons")
                                 .param("size", "1"))
                    .andExpect(status().isOk())
                    .andExpect(content().contentType(contentType))
-                   .andExpect(jsonPath("$._embedded.epersons", Matchers.contains(
-                       EPersonMatcher.matchDefaultTestEPerson()
-                   )))
-                   .andExpect(jsonPath("$._embedded.epersons", Matchers.not(
-                       Matchers.contains(
-                           EPersonMatcher.matchEPersonEntry(ePerson)
-                       )
-                   )))
+                        .andExpect(jsonPath("$._embedded.epersons", Matchers.contains(Matchers.anyOf(
+                       EPersonMatcher.matchEPersonOnEmail(eperson.getEmail()),
+                       EPersonMatcher.matchEPersonOnEmail(admin.getEmail()),
+                       EPersonMatcher.matchEPersonOnEmail(ePerson.getEmail())
```

Comment:
The variables eperson and ePerson are confusing in the same block.  Could more specific names be used?